scripts/bert/index.rst

```diff
@@ -195,6 +195,24 @@ To get the score of the dev data, you need to download the dev dataset (`dev-v2.
 
     $ python evaluate-v2.0.py dev-v2.0.json predictions.json
 
+BERT INT8 Quantization
+~~~~~~~~~~~~~~~~~~~~~~
+
+GluonNLP provides the following example scripts to quantize fine-tuned
+BERT models into int8 data type. Note that INT8 Quantization needs a nightly
+version of mxnet-mkl.
```

Comment:
@eric-haibin-lin Do we have any link about how to install nightly build from s3?

Reply:
The build wheels are here: https://apache-mxnet.s3-us-west-2.amazonaws.com/dist/index.html